pandas/core/dtypes/base.py

```diff
@@ -16,6 +16,10 @@ class _DtypeOpsMixin(object):
     # classes will inherit from this Mixin. Once everything is compatible, this
     # class's methods can be moved to ExtensionDtype and removed.
 
+    # na_value is the default NA value to use for this type. This is used in
+    # e.g. ExtensionArray.take.
```

Comment:
I think the "used in EA.take" is misleading. In many cases, it will not be used in take, as this `na_value` is your "boxed" scalar NA type, not the underlying physical NA value.